src/AcmTextInput/AcmTextInput.tsx

```diff
@@ -39,8 +47,31 @@ export function AcmTextInput(props: AcmTextInputProps) {
             hidden={props.hidden}
             helperTextInvalid={error}
             validated={validated}
+            helperText={helperText}
+            labelIcon={
+                /* istanbul ignore next */
+                props.labelHelp ? (
+                    <Popover
+                        id={`${props.id}-label-help-popover`}
+                        headerContent={labelHelpTitle}
+                        bodyContent={labelHelp}
+                    >
+                        <button
+                            id={`${props.id}-label-help-button`}
+                            aria-label="More info"
+                            onClick={(e) => e.preventDefault()}
+                            // aria-describedby="simple-form-name"
+                            className="pf-c-form__group-label-help"
+                        >
+                            <HelpIcon noVerticalAlign />
+                        </button>
+                    </Popover>
```

Comment:
Maybe a TODO here to support the Popover directly; and make this a little cleaner between all the former controls